App.js

```diff
@@ -1,22 +1,66 @@
 import { ApolloClient, ApolloProvider, InMemoryCache } from "@apollo/client";
-import { StatusBar } from "expo-status-bar";
+import { NavigationContainer } from "@react-navigation/native";
+import { createNativeStackNavigator } from "@react-navigation/native-stack";
 import React from "react";
-import { StyleSheet, View } from "react-native";
-import { SafeAreaProvider } from "react-native-safe-area-context";
-import { style } from "./App.style";
+import { StyleSheet, Text, View } from "react-native";
+import {
+  SafeAreaProvider,
+  initialWindowMetrics,
+} from "react-native-safe-area-context";
+import TodoDataLayer from "./views/Todo/Todo.dataLayer";
 import TodoListDataLayer from "./views/TodoList/TodoList.dataLayer";
+
+const Stack = createNativeStackNavigator();
+
+const linking = {
+  prefixes: ["http://localhost:19006"],
+  config: {
+    screens: {
+      TodoList: "/",
+      Todo: "/:id",
+    },
+  },
+};
+
 const client = new ApolloClient({
   uri: "https://flyby-router-demo.herokuapp.com/",
   cache: new InMemoryCache(),
 });
 
+function HomeScreen() {
+  return (
+    <View style={{ flex: 1, alignItems: "center", justifyContent: "center" }}>
```

Comment:
I'd recommend moving these styles out to a App.styles.js file in a stylesheet

Reply:
Oh this is just a test component. Should not have existed. Deleting.